content/3.pecan/1.overview/2.methods-and-data.md

```diff
@@ -153,20 +153,20 @@ Example data from 790 tumor samples can be found in [Figure 5 of McLeod et al](h
 
 ### t-SNE Methods
 
-[RAPID RNA-Seq workflow](https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/) was used to generate RNA-seq expression count data and visualized via ProteinPaint's scatter plot.
+St. Jude Cloud's [Rapid RNA-Seq workflow](https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/) was used to generate RNA-seq expression counts data and visualized on PeCan's expression t-SNE plot.
 
 ### t-SNE Data
 
 Sample data comes from G4K, PCGP, Clinical Pilot, and Real-Time Clinical Genomics studies.
 
 ::callout{icon="i-heroicons-light-bulb" color="blue"}
-**Tip**  
+**Tip**
 Example data from 1,574 RNA-Seq samples can be accessed [here](https://viz.stjude.cloud/st-jude-childrens-research-hospital/visualization/pediatric-blood-solid-and-brain-tumor-rna-seq-t-sne-plot-1574-samples~24).
 ::
 
 ### Gene Expression Methods
 
-DESeq2's Median of Ratios^1 was applied to ~5500 tumor samples for gene expression analysis.
+DESeq2's Median of Ratios was applied to ~5500 tumor samples for gene expression analysis.
```

Comment:
This needs to remove the citation from the `Key Benefits` list below. Though, we should keep the citation and just update the markdown so that it links correctly.